gapis/service/service.proto

```diff
@@ -1111,3 +1116,25 @@ message DeviceAPITraceConfiguration {
   // Does this API support MEC.
   FeatureStatus mid_execution_capture_support = 3;
 }
+
+message GetTimestampsRequest {
+  path.Capture capture = 1;
+  path.Device device = 2;
+}
+
+message TimestampsItem {
+  path.Command begin = 1;
+  path.Command end = 2;
+  uint64 time = 3;
```

Comment:
Especially important: What are the units of this time. 
I prefer to name things with their units
time_in_microseconds (for example) that way it is self-documenting.